x-pack/platform/plugins/shared/lens/public/visualizations/heatmap/toolbar_component/style_settings.tsx

```diff
@@ -110,13 +180,61 @@ export function HeatmapVerticalAxisSettings({
         }}
         isAxisLabelVisible={state?.gridConfig.isYAxisLabelVisible}
       />
+      <EuiFormRow
+        display="columnCompressed"
+        label={i18n.translate('xpack.lens.heatmap.sortOrder.label', {
+          defaultMessage: 'Sort order',
+        })}
+        fullWidth
+      >
```

Comment:
We can probably extract this into a component that can be reused in both x and Y